app/scripts/components/toolbar/arianne-thread.components.jsx

```diff
@@ -133,16 +138,13 @@ export default class ArianneThread extends React.Component {
 					toggleId="arianne-item-family"/>
 		);
 
-		const variants = variantFamily
-			? variantFamily.variants
-			: [];
 		const addVariant = <ArianneDropMenuItem item={{name: 'Add new variant...'}} click={this.addVariant}/>;
 		const variantItem = (
 				<DropArianneItem
 					label={this.state.variant.name}
 					family={this.state.family}
 					variant={this.state.variant}
-					list={variants}
+					list={this.state.family.variants.filter(({name}) => { return name !== this.state.variant.name; }) || []}
```

Comment:
Not sure for this one, since it's not defined in state (previously computed with `_.find` in `render()`).